baseline.yaml

```
@@ -649,6 +649,36 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-15
+    maturity_level: 2
+    category: Documentation
+    criteria: |
```

**nit:** This should be breaking CI, I'm confused why it doesn't 🤔

---

What do you mean?

---

@mlieberman85 we recently switch the yaml to use `criterion` instead of `criteria`. I believe that was merged after the last commit to this PR, which is why the CI hasn't failed as @blkt expects, but it should fail on the next commit if it's not fixed.

---

I gave this a re-run, no break... The unknown field just gets silently ignored.

We should fix it though.

```suggestion
    criterion: |
```